source/_integrations/lamarzocco.markdown

```diff
@@ -107,3 +116,34 @@ Host:
 |-------------|-------------| ------------------------| ---------------------- |
 | Prebrew/-infusion mode | Whether to use prebrew, preinfusion, or neither | Disabled, Prebrew, Preinfusion | Linea Micra, Linea Mini, GS3 AV |
 | Steam level | The level your steam boiler should run at | 1,2,3 | Linea Micra |
+
+## Supported devices
+Currently only devices from the `Home` range are supported. 
+
+That means
+
+- Linea Mini
+- Linea Mini R
+- Linea Micra
+- GS3 AV
+- GS3 MP
+
+## Possible use-cases
+- Control your machine through voice, allowing you to change boiler temperatures quickly without opening the app.
+- Control your smart coffee scales (tare/timer start) when a brew starts.
+- Turn on lights next to the machine while a brew is running.
+  
+## Known Limitations
+- Only La Marzocco native app accounts are supported, social logins (Google, Apple & WeChat) are not supported
+- Currently it is only possible to view the schedules configured in the La Marzocco Home app, but not to edit the schedules from Home Assistant. You can of course build Home Assistant native automations to reflect the same functionality in Home Assistant.
+
+## Troubleshooting
```

Comment:
```suggestion
## Troubleshooting

```

Reply:
@c0ffeeca7 Did we already settle on a standardized name for this section?

Reply:
yes, troubleshooting it is :tada: